pages/api/v1/latest-activity/on-chain-posts.ts

```diff
@@ -60,85 +61,106 @@ export async function getLatestActivityOnChainPosts(params: IGetLatestActivityOn
 			postsVariables.trackNumber_in = numTrackNo;
 		}
 
+		const query = network === AllNetworks.COLLECTIVES ? GET_ALLIANCE_LATEST_ACTIVITY : GET_PROPOSALS_LISTING_BY_TYPE;
```

Comment:
Handling for westendCollectives?

Reply:
This is an old code,
I removed that from this new branch [POL-256-updated-alliance](https://github.com/polkassembly/polkassembly/tree/POL-256-updated-alliance)
the reason is we are not using any on-chain-post function in the alliance because we have only 2 tabs one is All we have an all-post function for that and discussions that are off-chain-post